src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -290,15 +290,15 @@ export default class AircraftCommander {
         const turnDirection = data[0];
         const legLength = data[1];
         const holdFix = data[2];
-        let holdFixLocation = aircraft.position;
+        let holdPosition = aircraft.position;
         let inboundHdg = aircraft.heading;
 
         if (holdFix) {
-            holdFixLocation = this._navigationLibrary.getFixPositionCoordinates(holdFix);
-            inboundHdg = bearingToPoint(aircraft.position, holdFixLocation);
+            holdPosition = this._navigationLibrary.findFixByName(holdFix).position;
```

Comment:
`const { position: holdPosition } = this._navigationLibrary.findFixByName(holdFix);` this is a sort of confusing syntax, but I’m actually renaming a destructured property here this would be equivalent to `const holdPosition = this._navigationLibrary.findFixByName(holdFix).position;`  its just a shorter way of doing it.